### DISCUSSION TITLE
Make {@post_excerpt} work the same way the_excerpt does.

### DISCUSSION THREAD

**Discussion by m9n** [pods-framework/pods#6720] [Ideas]
Currently, there's no way with Pods shortcodes, to show a list of post excerpts, unless the user has explicitly set excepts for the posts.

WordPress calculates excerpts as follows (as far as I can tell):
1. Get the excerpt field from the database,
2. or, if that's not there, grab the text before the  tag in the post content,
3. or, if that's not there, grab a limited amount of text from the post content, based on a sane character/word count.

It would be really good (possibly even expected, by many) if the magic tag {@post_excerpt} worked in the same way.

**Comment by jimtrue** [comment]
I'd say this would have to be in 'addition' to, not instead of the way it works currently. post_content pulls the content 'untranslated' from post_content in the database, the same with post_excerpt (it pulls what is in the database for the post_excerpt.

As a workaround, have you tried writing your own php function that makes use of the excerpt logic from: https://developer.wordpress.org/reference/functions/the_excerpt/ ? Then you could call, `{@post_content,my_excerpt_function}`
